src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, user, body, comments } }) => (
```

Comment:
Props destructuring could be more explicit, it's better if you provide PropTypes to define the shape of the 'post' object.

Reply:
after installing the PropTypes module, the tests stopped working, so I did not make this fix